library/src/hipblas_module.f90

```diff
@@ -13236,4 +13236,85 @@ function hipblasZgeqrfStridedBatched(handle, m, n, A, lda, stride_A,&
         end function hipblasZgeqrfStridedBatched
     end interface
 
+    ! gels
+    interface
+        function hipblasSgels(handle, m, n, nrhs, trans, A, lda, B, ldb, info) &
+                result(c_int) &
+                bind(c, name = 'hipblasSgels')
```

Comment:
@cgmb brought to my attention that this is the wrong way to declare a Fortran result value. It goes beyond the scope of this PR, but you might want to consider making the same change as we made in hipSOLVER: https://github.com/ROCmSoftwarePlatform/hipSOLVER/pull/105.

Reply:
Yes not in this PR but we should make these fixes, there was an email chain where it was suggested we were going to pass this off to hipFortran but we should just go ahead and fix if you fixed solver.  Need to fix rocblas as well, I can do rocblas Daine if you will fix hipblas.

Reply:
Yes would't change just one function, we can look if we ever made a ticket, if not one lwp ticket for both hipblas and rocblas can suffice I would think.